web/app/models.py

```diff
@@ -226,7 +227,7 @@ def __str__(self):
 class TournamentPlayer(db.Model):
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     name = db.Column(db.String(80))
-    player_id = db.Column(db.String(5))
+    aga_num = db.Column(db.String(5))
     member_ex_date = db.Column(db.String(20))
     rating = db.Column(db.String(5))
```

Comment:
I was trying to remember...
I found this in the pair program doc:
"Programs should automatically assign temporary AGA numbers in the range 90000-99999."
But I'm not positive off hand. I may have just made a bad assumption based on the above.